types/koa-better-body/index.d.ts

```diff
@@ -25,6 +26,11 @@ declare namespace KoaBetterBody {
     decodeURIComponent?: any;
     maxKeys?: number;
     buffer?: boolean;
+    QueryString?: string;
```

Comment:
I believe this should just be `querystring`.

Reply:
Well, it's _querystring_. That's my mistake.

Reply:
Well, it's `querystring`. That's my mistake.